salt/pillar/rethinkdb_pillar.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+'''
+Provide external pillar data from RethinkDB
+
```

Comment:
Can you add `.. versionadded:: Oxygen` here?

Reply:
Done